CMakeLists.txt

```diff
@@ -67,16 +67,19 @@ if (${CMAKE_SYSTEM_NAME} STREQUAL "FreeBSD")
   TARGET_LINK_LIBRARIES(cminpack m)
 endif()
 
-# Link with a BLAS library if requested
-if (USE_BLAS)
-  if (NOT BUILD_SHARED_LIBS)
-    set(BLA_STATIC True)
-  endif()
-  find_package(BLAS)
-  if (BLAS_FOUND)
-    target_link_libraries(cminpack PUBLIC ${BLAS_LIBRARIES})
-    set_target_properties(cminpack PROPERTIES LINK_FLAGS "${BLAS_LINKER_FLAGS}")
-    target_compile_definitions(cminpack PUBLIC -DUSE_CBLAS)
+include (CheckLibraryExists)
```

Comment:
You should first check if sqrt exists without adding -lm. See https://github.com/awesomeWM/awesome/pull/984/files for a similar example